### PR TITLE
added new changes for the sidebar 

### PR DESCRIPTION
**Pull request by sumukhswamy**
### Description
added data connections tab in the sidebar

 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/components/Main/main.tsx [hunk 1]
```diff
@@ -683,27 +683,70 @@ export class Main extends React.Component<MainProps, MainState> {
         </div>
       );
     }
+    // const [isClearable, setIsClearable] = useState(true);
+
+    // const [value, setValue] = useState('');
+    // const onChange = (e) => {
+    //   setValue(e.target.value);
+    // };
```
Comment (ps48) [inline public/components/Main/main.tsx:691]: Can you please remove the commented code

File: public/components/Main/main.tsx [hunk 2]
```diff
@@ -3,10 +3,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTitle, EuiPageSideBar ,EuiSideNav, EuiPanel, EuiPageTemplate, EuiPage, EuiPageContent, EuiPageContentBody, EuiFlexGrid, EuiSplitPanel, EuiComboBox, EuiText, EuiPagination, EuiPopover, EuiFieldSearch} from '@elastic/eui';
 import { IHttpResponse } from 'angular';
 import _ from 'lodash';
-import React from 'react';
+import React, { useState }  from 'react';
```
Comment (ps48) [inline public/components/Main/main.tsx:9]: `useState` , `EuiPageTemplate` are unused header can you please remove them

**Review by ps48 — COMMENTED** [review]
You'll have to update the snapshot of jest tests when you're changing these files.

File: public/components/Main/main.tsx [hunk 3]
```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTitle, EuiPageSideBar ,EuiSideNav, EuiPanel, EuiPageTemplate, EuiPage, EuiPageContent, EuiPageContentBody, EuiFlexGrid, EuiSplitPanel, EuiComboBox, EuiText, EuiPagination, EuiPopover, EuiFieldSearch} from '@elastic/eui';
```
Comment (derek-ho) [inline public/components/Main/main.tsx:6]: Can you run lint here

**Pull request closed without merging**